pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -0,0 +1,61 @@
+package local
+
+import (
+	"context"
+	"github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"github.com/ibm/the-mesh-for-data/pkg/multicluster"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	m4dSystemNs                  string = "m4d-system"
+	clusterMetadataConfigmapName string = "cluster-metadata"
+)
+
+type ClusterManager struct {
```

Comment:
ClusterManager is already the interface name

Reply:
I tried before to call it `LocalClusterManager`, but Go's linter didn't like it and called it `stuttering`, and suggested to call it just `ClusterManager`. Bellow is a SOF question about it.https://stackoverflow.com/questions/45885985/ways-to-avoid-stuttering-in-go-package-and-struct-names

Reply:
I agree we need to be consistent. Another option is to call the interface `ClusterManagerInterface` or `IClusterManager`. @froesef  What do you think?

By the way, a name is considered to be stuttering if the package name is a strict prefix of it:
https://github.com/golang/lint/blob/master/lint.go#L944

Reply:
I was running into the same issue. I called it `MultiClusterManager` in dummy package. But I'm fine with `ClusterManager` and then the package name would define which one it is.

Reply:
@froesef I changed it to be `ClusterManager` in both implementations